src/translator/service.h

```diff
@@ -60,7 +60,7 @@ class Service {
   /// @param shortlistMemory byte array of shortlist (aligned to 64)
```

Comment:
The line above in documentation says `modelMemory` is Optional. Is it correct?

Reply:
This line is outdated. For this constructor, `modelMemory` is compulsory. I updated the documentation in the newest documentation.

Reply:
Cool. Thanks.